packages/db/src/translate/pdr.ts

```diff
@@ -52,5 +53,54 @@ export const translateApiPdrToPostgresPdr = async (
     created_at: (record.createdAt ? new Date(record.createdAt) : undefined),
     updated_at: (record.updatedAt ? new Date(record.updatedAt) : undefined),
   };
-  return pdrRecord;
+  return <PostgresPdr>removeNilProperties(pdrRecord);
+};
+
+/**
+ * Generate a Postgres PDR record from a DynamoDB record.
+ *
+ * @param {Object} postgresPDR - A Postgres PDR record
+ * @param {Object} knex - Knex client for reading from RDS database
+ * @param {Object} collectionPgModel - Instance of the collection database model
+ * @param {Object} providerPgModel - Instance of the provider database model
+ * @param {Object} executionPgModel - Instance of the execution database model
+ * @returns {Object} A PDR record
+ */
+export const translatePostgresPdrToApiPdr = async (
+  postgresPDR: PostgresPdr,
```

Comment:
Is there a valid reason this shouldn't be a PostgresPdrRecord?

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/eb35396fcd623ab9ea20b9450ad27b035ac1322d